src/components/articles/Helper.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component, Fragment } from 'react';
+
+class Helper extends Component {
```

Comment:
What does this component do? I think you should name it according to what it does.